docs/cni/rkt/cloud-config/master-config.yaml

```diff
@@ -1,21 +1,27 @@
 #cloud-config
 ---
 write_files:
-  - path: /home/core/get_rkt.sh
+  - path: /home/core/setup_calico.sh
     permissions: 777
     owner: root
     content: |
       #!/usr/bin/bash -e
-      # REMOVE WHEN COREOS SHIPS with rkt v0.16.0
       # Add /opt/bin to the _front_ of the PATH.
       # Can't directly write to .profile since it's a symlink to a RO filesystem      
       rm /home/core/.bashrc
       echo 'PATH=/opt/bin:$PATH' > /home/core/.bashrc
       echo "Defaults  secure_path=/opt/bin:$PATH" >>/etc/sudoers.d/path
       mkdir -p /opt/bin
-      curl -L https://github.com/coreos/rkt/releases/download/v0.16.0/rkt-v0.16.0.tar.gz | tar --strip-components=1 --overwrite -zxv -C /opt/bin
-      ln -s /opt/bin/stage1-fly.aci /home/core
-      ln -s /opt/bin/stage1-fly.aci /
+      mkdir -p /opt/bin
```

Comment:
Why move all of this into this script rather than leave it as ExecStartPre options?  Just curious.